src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -103,7 +103,7 @@ class WorkspaceMembersPage extends React.Component {
             this.setState((prevState) => ({
                 selectedEmployees: _.intersection(
                     prevState.selectedEmployees,
-                    _.keys(PolicyUtils.getClientPolicyMemberEmailsToAccountIDs(this.props.policyMembers, this.props.personalDetails)),
+                    _.map(_.keys(PolicyUtils.getClientPolicyMemberEmailsToAccountIDs(this.props.policyMembers, this.props.personalDetails)), (accountID) => Number(accountID)),
```

Comment:
I think the `keys` of what's returned should be emails, right?

Reply:
Ya i think it should be `_.values` - ya the function name is notttttttttt great 😳

Reply:
Should be fixed, the type error as well